node/cli/Cargo.toml

```diff
@@ -48,7 +48,7 @@ sp-inherents = { workspace = true }
 sp-keyring = { workspace = true }
 sp-runtime = { workspace = true }
 sp-timestamp = { workspace = true }
-try-runtime-cli = { workspace = true, optional = true }
```

Comment:
nit: Would you mind explaining why is this removed?

Reply:
Good question. I missed including that in the PR description. There’s now a standalone CLI, which is a result of [this Polkadot-sdk issue](https://github.com/paritytech/polkadot-sdk/pull/4017/files#diff-3a3aa5e088741f8ab1ca38fbe314c7a150d18b7466426f4ad2569113017e8439).